js/apps/admin-ui/src/components/group/GroupPickerDialog.tsx

```diff
@@ -224,7 +224,10 @@ export const GroupPickerDialog = ({
         <DataList aria-label={t("groups")} isCompact>
           {groups.slice(0, max).map((group: SelectableGroup) => (
             <Fragment key={group.id}>
-              {(!isSearching || group.name?.includes(filter)) && (
+              {(!isSearching ||
+                group.name
+                  ?.toLocaleUpperCase()
```

Comment:
Why use `.toLocaleUpperCase()` over `.toUpperCase()`? No locale is explicitly passed, nor would it change the amount of resulting matches.

Reply:
I assumed that .toLocaleUpperCase() might be able to detect the preferred locale (maybe from the browser?).  But I'm not sure if that's true.  It seems that if there is no other mechanism to detecting locale than passing a param then the no-arg version of the method should not be allowed, or at least detected by the linter.

Anyway, it doesn't make any difference.  Merging.